src/components/Header/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {
+  makeStyles, withStyles, createStyles, Theme,
+} from '@material-ui/core/styles';
+import {
+  AppBar, Toolbar, Typography, Button, Switch, Grid,
+} from '@material-ui/core';
+import Moon from './img/moon.png';
+import Sun from './img/sun.png';
+
+interface Props {
+  toggleTheme: (arg0: React.ChangeEvent<HTMLInputElement>) => void
+}
+
+const DarkModeSwitch = withStyles((theme: Theme) => createStyles({
+  root: {
+    width: 80,
+    height: 48,
+    padding: 10,
+  },
+  switchBase: {
+    padding: 9,
+    color: '#ff6a00',
+  },
+  thumb: {
+    width: 30,
+    height: 30,
+    backgroundColor: theme.palette.common.white,
+    border: 'solid',
+    borderColor: theme.palette.grey[800],
+  },
+  track: {
+    backgroundSize: '30px 28px',
```

Comment:
can we also make the track color a slightly darker one like the one in redux.js.org, also perhaps we can make the emoji not fill fit edge to edge, we can do something like `backgroundPosition: 'center', backgroundRepeat: 'no-repeat'` probably